### DISCUSSION TITLE
Unsupported by Crowdstrike?

### DISCUSSION THREAD

**Discussion by Derekt2** [CrowdStrike/falconpy#690] [Q&A] [answered]
I opened a support case for a bug I found in Crowdstrike API, and one of the first things the support rep mentioned was 
```
Falconpy is an open source tool available on Github for users that wish to interact with our APIs using Python.

Please note open source tools, such as Falconpy are not supported by Crowdstrike.
```
Is this accurate? Why not support a product which you maintain? Why would I use a product where if I have issues the first thing support does is ask me to replicate the bug using python's requests, curl, postman, etc. ?

**Comment by jshcodes** [comment, marked as answer]
Hi @Derekt2 -

Thank you for the question!

FalconPy is a CrowdStrike project, not a formal CrowdStrike product. This does **NOT** mean FalconPy is unsupported, just that [support](https://github.com/CrowdStrike/falconpy/blob/main/SUPPORT.md) for FalconPy should originate within the [issue tracking system](https://github.com/CrowdStrike/falconpy/issues) here within this repo. (This project's [maintainers](https://github.com/CrowdStrike/falconpy/blob/main/AUTHORS.md#maintainers), including myself, work for CrowdStrike.)

In regards to API issues discovered via FalconPy, feel free to report them here!  As part of our response, we will confirm the issue (which would involve testing using other tools such as `curl` when necessary). If we think the API is found to be the underlying root cause, we will report this immediately to the appropriate CrowdStrike team and will provide an update here. (Any followups we receive would also be posted here.)
